### PR TITLE
Update readme for ICL open source

### PR DESCRIPTION
**Pull request by Sherry-Lin**
add decode/encode/VP feature support for ICL
add known limitaion for kernel requirement

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,22 +115,22 @@ E2 - Low power encoding
...
+| Luma Key                                     |  Y  |  Y  |    Y    |  Y  |  Y  |  Y  |  Y  |
+| Mirroring                                    |  Y  |  Y  |    Y    |  Y  |  Y  |  Y  |  Y  |
+| ProcAmp (brightness,contrast,hue,saturation) |  Y  |  Y  |    Y    |  Y  |  Y  |  Y  |  Y  |
+| Rotation                                     |  Y  |  Y  |    Y    |  Y  |  Y  |  Y  |  Y  |
+| Scaling                                      |  Y  |  Y  |    Y    |  Y  |  Y  |  Y  |  Y  |
+| Sharpening                                   |  Y  |  Y  |    Y    |  Y  |  Y  |  Y  |  Y  |
+| STD/E (Skin Tone Detect & Enhancement)       |  Y  |  Y  |    Y    |  Y  |  Y  |  Y  |  Y  |
+| TCC (Total Color Control)                    |  Y  |  Y  |    Y    |  Y  |  Y  |  Y  |  Y  |
+| Color fill                                   |  Y  |  Y  |    Y    |  Y  |  Y  |  Y  |  Y  |
+| Chroma Siting                                |  N  |  Y  |    Y    |  Y  |  Y  |  Y  |  Y  |
```
Comment (dvrogozh) [inline README.md]: I thought that as of now we have upstreamed only Chroma Siting in VP, did not we?

Comment (Sherry-Lin) [reply README.md]: we up-streamed all of these features for ICL, confirmed with @FurongZhang

Comment (dvrogozh) [reply README.md]: cool, thank you.

File: README.md [hunk 2]
```diff
@@ -140,13 +142,15 @@ YV12/I420 as input format for csc/scaling/blending/rotation, etc. on Ubuntu 16.0
 (with kernel 4.10). The issue can be addressed with the kernel patch:
 WaEnableYV12BugFixInHalfSliceChicken7 [commit 0b71cea29fc29bbd8e9dd9c641fee6bd75f6827](https://cgit.freedesktop.org/drm-tip/commit/?id=0b71cea29fc29bbd8e9dd9c641fee6bd75f68274)
 
-3. CNL: HEVC encoding does not support P frame.
+3. KBL/CFL/CNL: HEVC encoding does not support P frame, detail see [issue#206](https://github.com/intel/media-driver/issues/206)/[issue#207](https://github.com/intel/media-driver/issues/207)/[issue#209](https://github.com/intel/media-driver/issues/209), patch in [PR#262](https://github.com/intel/media-driver/pull/262).
```
Comment (dvrogozh) [inline README.md]: Suggest to slightly reformulate: "KBL/CFL/CNL: HEVC encoding does not support P frame. For details see [issue#206](https://github.com/intel/media-driver/issues/206)/[issue#207](https://github.com/intel/media-driver/issues/207)/[issue#209](https://github.com/intel/media-driver/issues/209). Patches under review: [PR#262](https://github.com/intel/media-driver/pull/262"

File: README.md [hunk 3]
```diff
@@ -140,13 +142,15 @@ YV12/I420 as input format for csc/scaling/blending/rotation, etc. on Ubuntu 16.0
...
 
 4. HuC firmware is needed for AVC low power encoding bitrate control, including CBR, VBR, etc. As of now, HuC firmware support is disabled in Linux kernels by default. Please, refer to i915 kernel mode driver documentation to learn how to enable it. Mind that HuC firmware support presents in the following kernels for the specified platforms:
    * APL, KBL: starting from kernel 4.11
    * CFL: starting from kernel 4.15
 
 5. Restriction in implementation of vaGetImage: Source format (surface) should be same with destination format (image).
 
+6. ICL encoding and decoding require ICL special kernel mode driver ([issue#267](https://github.com/intel/media-driver/issues/267)/[PR#271](https://github.com/intel/media-driver/pull/271)), please refer to i915 kernel mode driver documentation for supporting status.
```
Comment (dvrogozh) [inline README.md]: "ICL encoding and decoding require special kernel mode driver" - drop 2nd "ICL"...

**Comment by Sherry-Lin** [comment]
Updated. Thanks for your comments.

**Pull request merged**